_authors/davidlago.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Dave Lago
+short_name: davidlago
+photo: '/assets/media/authors/davidlago.jpg'
+github: davidlago
+---
+
+**Dave Lago** is a Software Development Manager at AWS working on the OpenSearch project.
```

Comment:
This might be smoother as "Dave Lago is an AWS Software Development Manager working on the OpenSearch project."